src/commands/moderation/repel.ts

```diff
@@ -442,6 +498,17 @@ export const repelCommand = createCommand({
         lookBack: lookBack ?? DEFAULT_LOOK_BACK_MS,
       });
 
+      const shouldDMUser = interaction.options.getBoolean(RepelOptions.DM_USER) ?? true;
+      let dmSent = false;
+      if (shouldDMUser) {
+        dmSent = await sendReasonToTarget({
+          target,
+          reason,
+          guildName: interaction.guild.name,
+          timeoutDuration: timeout,
+        });
+      }
```

Comment:
and then (2/2) we do this here
```suggestion
      if (shouldDMUser) {
      try {
        await sendReasonToTarget({
            target,
            reason,
            guildName: interaction.guild.name,
            timeoutDuration: timeout,
          });
        }
        dmSent = true; // but could also be default to true
      } catch {
        dmSent = false;
      }
```